docs/pushlog.md

```diff
@@ -8,7 +9,10 @@ closely mirrors the logic treeherder itself uses but is duplicated here
 to ensure that the moment we notice a push we create a resultset and the
 associated graph.
 
-The Repositories collection contains the list of monitored repositories
-(Note that this only is supported for repositories which live under the
-hg.mozilla.org/* host with the pushlog extension.
+The Repositories collection in MongoDB contains the list of monitored
+repositories Note that this only is supported for repositories which live
```

Comment:
I forgot to push this change before clicking "merge" so I landed this fix on master.